### PR TITLE
WIP finshished MVP

### PR DESCRIPTION
**Pull request by trucane**

### CODE REVIEW

File: assignments/lambda-classes.js [hunk 1]
```diff
@@ -1 +1,130 @@
 // CODE here for your Lambda Classes
+
+class Person {
+    constructor({name, age, location}){
```
Comment (asingh714) [inline assignments/lambda-classes.js:4]: great work in pushing yourself to destructure your objects

File: assignments/lambda-classes.js [hunk 2]
```diff
@@ -1 +1,130 @@
...
+  console.log(aman.catchPhrase)
+  console.log(fred.graduate());
+  console.log(dan.demo('javascript'));
+  console.log(dan.grade(fred, 'CSS Fundamentals'));
+  console.log(fred.grade);
+  console.log(aman.score(fred));
+  console.log(fred.listSubjects());
+  console.log(barney.PRAssignment('Javascript II'));
+  console.log(barney.sprintChallenge('Advanced CSS'));
+  console.log(aman.standUp('Web_20_Holla_Holla'));
```
Comment (asingh714) [inline assignments/lambda-classes.js:128]: hahaha

File: assignments/prototype-refactor.js [hunk 3]
```diff
@@ -7,3 +7,241 @@ Prototype Refactor
...
+  
+   console.log(villan.destruction()); //Destroying a city
+   console.log(hero.heroMontra()); // Has arrived
+   //console.log(hero.BlazeofGlory()); // Has arrived
+  
+   
+  
+  
+  
+    // Stretch task:
```
Comment (asingh714) [inline assignments/prototype-refactor.js:220]: Were you able to finish this?

**Review by asingh714 — COMMENTED** [review]
Great work on the MVP and stretch

**Comment by trucane** [comment]
No not yet. Its Prom day, things been hectic since last night

**Pull request merged**